Automattic/wp-calypso
# Reader: send all posts across the redux bridge

Part of the reduxification effort of #10984  and dependent on #22110.

Posts are pretty tricky and touch _all_ of Reader.  My thought is to tackle this in two layers:
1. Data Storage + requests to fetch posts. I plan on replacing the `PostStore.get` functionality.  Redux should be the only place to look for posts.  If a component needs a post, then it can use a query component that secretly uses flux under the hood.
2. (future PRs): Data fetching --> data layer

---
**To Test**
- run this brach locally. click around reader. You should see `state.reader.posts.items` populate with many posts! When you refresh it should be empty.
- open all of our streams, liking posts, searching for posts, recs, combined cards, everything.
- open up RelatedPosts and click into more RPs.  Especially for sites like Time and Uproxx
- use the `at` link and click an update pill. then click a gap
